rdf_differ/utils.py

```diff
@@ -0,0 +1,67 @@
+"""
+utils.py
```

Comment:
Usually, when you have such a module in a project, after a while, you end up with a bunch of crap in it. Maybe it's better to give this proper name? IDK, `file_utils.py` so that you will keep file related things? Or you can find a better name 😄